pkg/ddc/jindocache/transform.go

```diff
@@ -275,17 +275,22 @@ func (e *JindoCacheEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, m
 			cachesetPath = "local://" + ufsVolumesPath
 		}
 		cacheStrategy := "DISTRIBUTED"
-		metaPolicy := "ALWAYS"
+		metaPolicy := "ONCE"
 		readCacheReplica := 1
 		writeCacheReplica := 1
 
-		if mount.Options["cacheStrategy"] == "DHT" && mount.Options["metaPolicy"] == "ONCE" {
-			cacheStrategy = mount.Options["cacheStrategy"]
+		if mount.Options["metaPolicy"] == "ALWAYS" {
 			metaPolicy = mount.Options["metaPolicy"]
 		}
 
-		if mount.Options["metaPolicy"] == "ONCE" {
-			metaPolicy = mount.Options["metaPolicy"]
+		if mount.Options["cacheStrategy"] == "DHT" && metaPolicy == "ONCE" {
```

Comment:
There are two checks for mount.Options["cacheStrategy"] == "DHT" in the code. How about using the following way:

```golang
if cacheStrategy := mount.Options["cacheStrategy"]; cacheStrategy == "DHT" {
    if metaPolicy == "ONCE" {
        cacheStrategy = cacheStrategy 
    } else {
        err = fmt.Errorf("CacheStrategy DHT must be used with metaPolicy ONCE and current metaPolicy is %s", metaPolicy)
        e.Log.Error(err, "Incorrect metaPolicy", metaPolicy)
        return err
    }
}
```